src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -408,6 +408,42 @@ public final BeanDescription introspectDirectClassAnnotations(JavaType type) {
     }
 
     /**
+     * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+     * possible per-type override for given property type.<br>
+     * NOTE: if no override found, defaults to value returned by
+     * {@link #getDefaultPropertyInclusion()}.
+     *
+     * @since 2.9
+     */
+    public abstract JsonInclude.Value getDefaultInclusion(Class<?> baseType,
```

Comment:
Would be good to add brief explanation regarding difference between `baseType`, `propertyType`, in context of properties of given POJO.